tests/robot-tests/tests/libs/admin-common.robot

```diff
@@ -310,3 +310,148 @@ user enters text into meta guidance data file content editor
     user clicks element  ${editor}
     user waits until page contains testid   fileGuidanceContent-focused
     user enters text into element  ${editor}  ${text}
+
+
+user adds summary text block to content section
+    [Arguments]  ${text}
+    user clicks button  Add a summary text block 
+    user clicks button  Edit block 
+    user enters text into element  //*[@role="textbox"]//p  ${text} 
+    user clicks button  Save
+
+
+user adds release note to content 
+    [Arguments]  ${text}
+    user clicks button  Add note
+    user enters text into element  //*[@id="createReleaseNoteForm-reason"]  ${text}
+
+user creates amendment for release
+    [Arguments]  ${PUBLICATION_NAME}  ${RELEASE_NAME}  ${RELEASE_STATUS}
+    user goes to url  %{ADMIN_URL}
+    user selects theme and topic from admin dashboard  %{TEST_THEME_NAME}  %{TEST_TOPIC_NAME}
+    user waits until page contains accordion section   ${PUBLICATION_NAME}
+    user opens accordion section  ${PUBLICATION_NAME}
+    ${accordion}=  user gets accordion section content element  ${PUBLICATION_NAME}
+    user opens details dropdown   ${RELEASE_NAME} ${RELEASE_STATUS}  ${accordion}
+    ${details}=  user gets details content element  ${RELEASE_NAME} ${RELEASE_STATUS}  ${accordion}
+    user waits until parent contains element   ${details}   xpath:.//a[text()="View this release"]
+    user clicks button  Amend this release  ${details}
+    user clicks button  Confirm
+
+user replaces data files for amendment 
+    [Arguments]  ${SUBJECT_NAME}  ${SUBJECT_FILE}  ${META_FILE}  ${text}  ${ERROR}
+    [Documentation]  EES-1442
+    user clicks link  Data and files
+    user waits until page contains element  id:dataFileUploadForm-subjectTitle
+    user waits until h2 is visible  Uploaded data files
+    user waits until page contains accordion section   ${SUBJECT_NAME}
+    user opens accordion section   ${SUBJECT_NAME}
+    ${section}=  user gets accordion section content element  ${SUBJECT_NAME}
+    user clicks link  Replace data  ${section}
+    user chooses file   id:dataFileUploadForm-dataFile       ${FILES_DIR}${SUBJECT_FILE}
+    user chooses file   id:dataFileUploadForm-metadataFile   ${FILES_DIR}${META_FILE}
+    user clicks button  Upload data files
+    user clicks button  Confirm
+    #EES-1442: Bug when Confirm data replacement button doesn't show
+    user reloads page
+    user waits until page contains  Footnotes: OK
+    user waits until page contains  Data blocks: OK
+    user waits until button is enabled  Confirm data replacement
+
+user deletes subject file 
+    [Arguments]  ${SUBJECT_NAME}
+    user clicks link  Data and files
```

Comment:
I think I'd remove `user clicks link Data and files` from this keyword and put it in the test suite itself, so its clearer to people reading the tests what is going on.

If a keyword is doing the navigating somewhere as part of what it does, then it should be included in the name of the keyword. People reading and writing the tests, in an ideal world, wouldn't have to click through to see what the keyword is doing. If some keywords do navigate and some don't, and the name doesn't indicate whether it does, then the user is forced to click through.